scenarios/microsoft-azure-arcbox-v1/discovery.metadata.json

```diff
@@ -1,21 +1,19 @@
 {
-  "icon": null,
+  "icon": "Microsoft.Arc/all",
   "title": "Get hands-on with ArcBox",
-  "shortDescription": "This sample scenario template helps illustrate how the metadata will show up in solutions center",
+  "shortDescription": "Use ArcBox to deploy an Azure Arc sandbox in less than an hour.",
   "cost": null,
-  "activationString": "Click to view details of the Solution",
+  "activationString": "View details of the solution",
   "keywords": ["Sample","solution", "template"],
-  "detailsDescription": "Soe more contextual details about what happens with this solution and why this is useful and why a user should use it.",
-  "learnMoreLink": "aka.ms/learnmoreaboutme",
-  "learnMoreLinkText": "Learn more about Sample solution",
+  "detailsDescription": "See more contextual details about what happens with this solution and why this is useful and why a user should use it.",
```

Comment:
"...and why you would use it " perhaps ? You are directly addressing the user here.

Reply:
i don't know where this is displayed, i just left the default and fixed "Soe" to "See"